fastapi_pagination/types.py

```diff
@@ -14,6 +15,7 @@
 Cursor: TypeAlias = Union[str, bytes]
 ParamsType: TypeAlias = Literal["cursor", "limit-offset"]
 AdditionalData: TypeAlias = Optional[Dict[str, Any]]
+ItemsTransformer: TypeAlias = Callable[[Sequence[Any]], Sequence[Any]]
```

Comment:
Does this allow the transformer to be async?  This would be important to be able to populate the paged items with additional data from the DB, and external API, etc.

Reply:
Thanks for pointing it out, fixed